app/models/user.rb

```diff
@@ -3,9 +3,32 @@ class User < ApplicationRecord
   include Comfy::Cms::WithCategories
   # Include default devise modules. Others available are:
   #  and :omniauthable
-  devise :invitable, :database_authenticatable, :registerable,
+  devise :invitable, :registerable,
          :recoverable, :rememberable, :validatable,
-         :confirmable, :lockable, :timeoutable, :trackable
+         :confirmable, :lockable, :timeoutable, :trackable,  :two_factor_authenticatable,
+         :otp_secret_encryption_key => ENV['ENCRYPTION_KEY']
```

Comment:
@ajaystha42 use rails secret

Reply:
I have used like this:          :otp_secret_encryption_key => ENV['RAILS_SECRET']. is it okay?

Reply:
@ajaystha42 use the secrets API: https://apidock.com/rails/Rails/Application/secrets -- don't read off the ENV since it may not be set like that depending on the environment

Reply:
okay